### ISSUE TITLE
[Enhancement]: File search function

### ISSUE THREAD

**Issue by nothingman4ik** [AssetRipper/AssetRipper#1460]
### Describe the new feature or enhancement

When you really need to find a specific file but there is no search and you have to do it manually every time

**Comment by ds5678** [comment]
Do you mean search for a specific asset?

**Comment by TNXG** [comment]
Yes, I'm not sure if I missed it or not, but I didn't find a section in AssetRipper for searching specific content. This makes it really painful to search for specific resources in large projects. For example, I could search by container path to get all relevant information about the content.

**Comment by Ikkoru** [comment]
A search function for text in all the files of a project would be nice too.
